check/bench_test.go

```diff
@@ -0,0 +1,143 @@
+package check
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+
+const check = `---
+controls:
+id: 1
+text: "Master Node Security Configuration"
+type: "master"
+groups:
+- id: 1.1
+  text: "API Server"
+  checks:
+    - id: 1.1.1
+      text: "Ensure that the --allow-privileged argument is set to false (Scored)"
+      audittype: "wrong_type"
+      audit: "ps -ef | grep $apiserverbin | grep -v grep"
+      tests:
+        test_items:
+        - flag: "allow-privileged"
+          compare:
+            op: eq
+            value: false
+          set: true
+      remediation: "Edit the $apiserverconf file on the master node and set 
+              the KUBE_ALLOW_PRIV parameter to \"--allow-privileged=false\""
+      scored: true
+`
+
+
+func Test_bench_RegisterAuditType_Already_Registered(t *testing.T) {
```

Comment:
Not a big deal, but I’d just call this `TestRegisterAuditType` (the detailed name is actually misleading because it tests the normal registration path as well as the already-registered case). Also see https://golang.org/pkg/testing/ - it’s nice to use the naming convention so you can do things like `go test -run Foo`